### PR TITLE
Fix fragile replica set failover tests for 3.0

### PR DESCRIPTION
**Pull request by jmikola**

### CODE REVIEW

File: tests/replicaset-failover/mongoinsertbatch-execute-002.phpt [hunk 1]
```diff
@@ -52,7 +52,7 @@ echo "Test completed\n";
 Killing master after 100
 Killing done
 Restarting done
-MongoCursorException: localhost:30200: Remote server has closed the connection
+MongoCursorException: localhost:%d: Remote server has closed the connection
```
Comment (bjori) [inline tests/replicaset-failover/mongoinsertbatch-execute-002.phpt]: 30200 is the highest priority member -- how did it not be primary? o.O

Comment (jmikola) [inline tests/replicaset-failover/mongoinsertbatch-execute-002.phpt]: @derickr and I confirmed that yesterday. It certainly is odd.

Perhaps we should hold off on merging this for a bit.

**Comment by jmikola** [comment]
Freshly rebased on `v1.6`.

**Comment by jmikola** [comment]
No need to merge this since the project is EOL.

**Pull request closed without merging**